src-docs/src/views/markdown_editor/markdown_editor_with_plugins.js

```diff
@@ -246,7 +248,7 @@ const ChartMarkdownRenderer = ({ palette, categories }) => {
   return (
     <Chart size={{ height: 320 }}>
       <Settings
-        theme={[customColors]}
+        theme={[customColors, OUI_CHARTS_THEME_LIGHT]}
```

Comment:
Note to self - open issue to look into migrating/removing elastic chart usage from OUI itself.